src/implementations/twig/components/blockquote/blockquote.story.js

```diff
@@ -40,10 +41,28 @@ const getArgTypes = () => {
         type: 'text',
       },
     },
+    show_image: {
+      name: 'show_image',
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Show image',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
+        category: 'Content',
+      },
+    },
   };
 };
 
 const prepareData = (data, args) => {
+  if (args.show_image) {
+    data.image = {
+      path: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image-square.jpg',
```

Comment:
well, like this if we change the image again the specs you also need to update it here.
You can clone this value from the data object into a constant that then you can use here, instead.